collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/BackgroundAudioRecordingTest.java

```diff
@@ -106,8 +109,35 @@ public void fillingOutForm_recordsAudio() {
         formEndPage.clickSaveAndExit();
         assertThat(stubAudioRecorderViewModel.isRecording(), is(false));
 
-        assertThat(stubAudioRecorderViewModel.getLastRecording(), notNullValue());
-        assertThat(stubAudioRecorderViewModel.getLastRecording().exists(), is(true));
+        File instancesDir = new File(testDependencies.storagePathProvider.getDirPath(StorageSubdirectory.INSTANCES));
```

Comment:
I think I'd rather be testing this at a lower level  but there I'm not that confident that the code that handles recordings is going to stay where it is, so it felt safer (and was easier) to drive the feature out from the top level.

Reply:
As discovered in https://github.com/getodk/collect/pull/4359#issuecomment-774397604, we just have to be careful about the fact that this will always use unscoped storage so there could be some serious bugs it doesn't help notice.